string bone_type
float64[3] start_position
float64[3] end_position
float64[3] direction